--- nget-0.27.1.orig/myregex.cc	2004-05-30 04:30:06.000000000 +0530
+++ nget-0.27.1/myregex.cc	2008-02-16 21:55:45.000000000 +0530
@@ -19,7 +19,8 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
-#include <stdlib.h>
+#include <cstdlib>
+#include <cstring>
 #include "myregex.h"
 

--- nget-0.27.1.orig/nrange.h	2004-05-30 04:30:08.000000000 +0530
+++ nget-0.27.1/nrange.h	2008-02-16 21:55:45.000000000 +0530
@@ -22,6 +22,7 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
+#include <climits>
 #include <sys/types.h>
 #include <map>
 #include <stdexcept>
diff -urNp nget-0.27.1.orig/path.cc nget-0.27.1/path.cc
--- nget-0.27.1.orig/path.cc	2004-05-30 04:30:06.000000000 +0530
+++ nget-0.27.1/path.cc	2008-02-16 21:55:45.000000000 +0530
@@ -17,7 +17,8 @@
     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 */
 #include "path.h"
-#include <errno.h>
+#include <cerrno>
+#include <cstring>
 #include <unistd.h>
 #include "log.h"
 #include "_fileconf.h"
diff -urNp nget-0.27.1.orig/server.cc nget-0.27.1/server.cc
--- nget-0.27.1.orig/server.cc	2004-12-21 02:14:11.000000000 +0530
+++ nget-0.27.1/server.cc	2008-02-16 22:09:43.000000000 +0530
@@ -20,6 +20,7 @@
 #include "strreps.h"
 #include "nget.h"
 #include "status.h"
+#include <climits>
 #include <sys/types.h>
 #include <dirent.h>
 #include <algorithm>
diff -urNp nget-0.27.1.orig/stlhelp.h nget-0.27.1/stlhelp.h
--- nget-0.27.1.orig/stlhelp.h	2004-05-30 04:30:08.000000000 +0530
+++ nget-0.27.1/stlhelp.h	2008-02-16 21:55:45.000000000 +0530
@@ -19,6 +19,7 @@
 #ifndef _STLHELP_H_
 #define _STLHELP_H_
 
+#include <cstring>
 struct eqstr
 {
 	bool operator()(const char* s1, const char* s2) const
