diff -urN nget/myregex.cc nget-patched/myregex.cc
--- nget/myregex.cc	2008-03-03 17:03:02.000000000 +1000
+++ nget-patched/myregex.cc	2014-03-20 20:49:15.398947964 +1100
@@ -19,7 +19,8 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
-#include <stdlib.h>
+#include <cstdlib>
+#include <cstring>
 #include "myregex.h"
 #include <string.h>
 
diff -urN nget/nrange.h nget-patched/nrange.h
--- nget/nrange.h	2008-03-03 17:03:02.000000000 +1000
+++ nget-patched/nrange.h	2014-03-20 20:49:15.398947964 +1100
@@ -22,6 +22,7 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
+#include <climits>
 #include <sys/types.h>
 #include <map>
 #include <stdexcept>
diff -urN nget/path.cc nget-patched/path.cc
--- nget/path.cc	2008-03-03 17:03:02.000000000 +1000
+++ nget-patched/path.cc	2014-03-20 20:49:15.398947964 +1100
@@ -17,7 +17,8 @@
     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 */
 #include "path.h"
-#include <errno.h>
+#include <cerrno>
+#include <cstring>
 #include <unistd.h>
 #include "log.h"
 #include "_fileconf.h"
diff -urN nget/server.cc nget-patched/server.cc
--- nget/server.cc	2005-05-12 15:32:50.000000000 +1100
+++ nget-patched/server.cc	2014-03-20 20:49:15.398947964 +1100
@@ -20,6 +20,7 @@
 #include "strreps.h"
 #include "nget.h"
 #include "status.h"
+#include <climits>
 #include <sys/types.h>
 #include <dirent.h>
 #include <algorithm>
diff -urN nget/stlhelp.h nget-patched/stlhelp.h
--- nget/stlhelp.h	2008-03-03 17:03:02.000000000 +1000
+++ nget-patched/stlhelp.h	2014-03-20 20:49:15.399947981 +1100
@@ -21,6 +21,7 @@
 
 #include <string.h>
 
+#include <cstring>
 struct eqstr
 {
 	bool operator()(const char* s1, const char* s2) const
